docs/site/Using-strong-error-handler.md

```diff
@@ -0,0 +1,251 @@
+[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
+package.
+
+# strong-error-handler
+
+This package is an error handler for use in both development (debug) and
+production environments.
+
+## Supported versions
+
+| Current |
+| :-----: |
+|   3.x   |
+
+## Installation
+
+```bash
+$ npm install --save strong-error-handler
+```
+
+## Usage
+
+As mentioned above, errors are handled by the `reject` action of the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html). By default,
+implementation of `reject`, `strong-error-handler` omits details from error
+responses to prevent leaking sensitive information.
+
+`strong-error-handler` also can return full error stack traces and internal
+details of any error objects to the client in the HTTP responses once the
+`debug` flag is set. Please check
+[Sequence - Handling errors](Sequence.md#handling-errors) section for usages and
+examples.
+
+The module also exports `writeErrorToResponse`, a non-middleware flavor of the
+error handler:
+
+```js
+import {Server} from 'http';
+import {writeErrorToResponse} from 'strong-error-handler';
+const errHandlingOptions = {debug: true, log: true};
+
+http
+  .createServer(function handleRequest(req, res) {
+    if (errShouldBeThrown) {
+      writeErrorToResponse(
+        new Error('something went wrong'),
+        req,
+        res,
+        errHandlingOptions,
+      );
+    }
+  })
+  .listen(3000);
+```
+
+In general, `strong-error-handler` must be the last middleware function
+registered.
+
+The above configuration will log errors to the server console, but not return
+stack traces in HTTP responses. For details on configuration options, see below.
+
+### Working with Express middleware
+
+Under the hood, LoopBack leverages [Express](https://expressjs.com) framework
+and its concept of middleware. To avoid common pitfalls, it is not possible to
+mount Express middleware directly on a LoopBack application.
+
+`strong-error-handler` can be bond to the app, which is the same as above. For
+example:
+
+```ts
+app.bind(RestBindings.ERROR_WRITER_OPTIONS).to({debug: true});
+```
+
+### Response format and content type
+
+The `strong-error-handler` package supports JSON, HTML and XML responses:
+
+- When the object is a standard Error object, it returns the string provided by
+  the stack property in HTML/text responses.
+- When the object is a non-Error object, it returns the result of `util.inspect`
+  in HTML/text responses.
+- For JSON responses, the result is an object with all enumerable properties
+  from the object in the response.
+
+The content type of the response depends on the request's `Accepts` header.
+
+- For Accepts header `json` or `application/json`, the response content type is
+  JSON.
+- For Accepts header `html` or `text/html`, the response content type is HTML.
+- For Accepts header `xml` or `text/xml`, the response content type is XML.
+
+_There are plans to support other formats such as Plain-text._
```

Comment:
should it be `text/plain`? 
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types.